src/nvidia-cuda/install.sh

```diff
@@ -17,6 +17,11 @@ if [ "$(id -u)" -ne 0 ]; then
     exit 1
 fi
 
+if [[ $(uname -m) = "aarch64" ]]; then
+    echo -e "NVIDIA® CUDA Toolkit no longer supports development or running applications on machines with the 'arm64' architecture. Please consider an alternative or check for updates or support for your architecture."
+    exit 1
```

Comment:
> Note that, I have other developers on my repository who are using containers with a classic chip architecture, and it's essential for me to retain this feature for them.
> My primary question is: How can I add a condition to specify the architecture and install or not the feature?

From the discussion, note that if the Feature returns exit code `1`, then your dev container creation will fail. Unless, you switch the Feature option `installToolkit` to `false`.